### PR TITLE
Add OSTree cheat sheet in README

### PR DESCRIPTION
**Pull request by atodorov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,3 +103,60 @@ Produce code coverage report
...
+OSTree repository. Here are a few quick examples how to work with `ostree`:
+
+
+To see what references aka "branches" are in the repo:
+
+    ostree --repo=cs.repo/ refs
+    master
+
+
+To see the commit log (use less, ostree doesn't paginate like git):
```
Comment (clumens) [inline README.md]: Usually I'm not picky with spelling, but this is docs so I guess I will be.  "use less" is one word - "useless".

File: README.md [hunk 2]
```diff
@@ -103,3 +103,60 @@ Produce code coverage report
...
+    -00644 0 0   1892 835ffd08dba662f47c7eb3164ae76aa5903363c63d3d606268c0ba928b725e6e /usr/share/doc/zsh-html-5.0.2/Alternate-Forms-For-Complex-Commands.html
+    -00644 0 0   1830 59184f18d807a6b9fc963b709ef82dba4d00f536388935aa79b74160fe90cbfe /usr/share/doc/zsh-html-5.0.2/Alternative-Completion.html
+    -00644 0 0   1734 b1bd5813c780a0862843fd3f468021c9358f5852c8a2029cb6857fb64dd2aed1 /usr/share/doc/zsh-html-5.0.2/Arguments.html
+    -00644 0 0  17635 89309b924ef3874172c73bab3c45eab56da0b847d6328dc387e0e1c5b3dfa633 /usr/share/doc/zsh-html-5.0.2/Arithmetic-Evaluation.html
+
+**NOTE:** `-R` means recursive, `-C` displays file checksums
+
+To check out aka export a commit into a filesystem tree:
+
+    sudo ostree --repo=cs.repo/ checkout 816b63ab93a7a866d598c552f212c3a407648096521fde74cc9aa317730da8b1 some.dir/
```
Comment (clumens) [inline README.md]: You really have to be root to do this?  That's too bad.

File: README.md (hunk 1 above)
Comment (atodorov) [inline README.md]: I mean use the `less` command. Any preferences over more verbose wording or using backticks ?

File: README.md (hunk 2 above)
Comment (atodorov) [inline README.md]: I guess this is how import creates the repository. Doesn't work without being root/

**Pull request merged**